apps/wallet-mobile/src/features/Receive/common/AddressDetailCard/AddressDetailCard.tsx

```diff
@@ -61,17 +65,24 @@ export const AddressDetailCard = ({address, title, addressDetails}: ShareProps)
     setScrollPosition(index)
   }
 
+  if (selectedAddress === undefined) {
+    return
+  }
+
   const renderItem = ({item}: {item: CardItem}) => {
     switch (item.cardType) {
       case 'QRCode':
-        return (
-          <ShareQRCodeCard
-            title={item.title}
-            address={item.address}
-            onLongPress={() => copy(mocks.address)}
-            isCopying={isCopying}
-          />
-        )
+        if (selectedAddress !== undefined) {
```

Comment:
this should affect only the QR code? or should never happen? Cuz Details is render anyways.

Reply:
can remove it I think